server/store/migrations/0001_initial.py

```diff
@@ -1,30 +1,17 @@
-#!/usr/bin/python
-#
-# Copyright 2022 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
+# Generated by Django 4.2 on 2023-04-20 03:38
```

Comment:
question: Let me walk through what I see, please confirm I understood!

1. Deleting two previously existing migrations
2. Reusing the 0001 migration
3. Aliasing the now deleted migrations to the new migration so existing deployments recognize no migration is needed
4. We've kept the 0001 numbering so folks that look at the Avocano code to learn about database migrations don't need to wonder why we are on 0003.

Reply:
I also realise I dropped the licence header here, will restore.